modal-window.js

```diff
@@ -0,0 +1,136 @@
+const cardsContainer = document.getElementById('work-section');
+
+const cards = [
+  {
+    img: 'assets/1.png',
+    project_title: 'TONIC',
+    company: 'CANOPY',
+    dev_title: 'BackEndDev',
+    date: '2015',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Libero sapiente quod corporis voluptate praesentium adipisci nisi fuga quaerat.',
+    skills: ['html', 'Ruby on rails', 'css', 'Javascript'],
+  },
+  {
+    img: 'assets/2.png',
+    project_title: 'TONIC',
+    company: 'CANOPY',
+    dev_title: 'BackEndDev',
+    date: '2015',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Libero sapiente quod corporis voluptate praesentium adipisci nisi fuga quaerat.',
+    skills: ['html', 'Ruby on rails', 'css', 'Javascript'],
+  },
+  {
+    img: 'assets/3.png',
+    project_title: 'TONIC',
+    company: 'CANOPY',
+    dev_title: 'BackEndDev',
```

Comment:
- [ ] [OPTIONAL] It will be better for your project when you click in see project to see the same image for the project not static one to each model, so in future when you need to add the project just to add data for the project in cards array which should display it dynamically.